lib/PuppeteerSharp.Tests/Page/MetricsTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Page
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class MetricsTests : PuppeteerPageBaseTest
+    {
+        [Fact]
+        public async Task ShouldGetMetricsFromPage()
+        {
+            await Page.GoToAsync("about:blank");
+            var metrics = await Page.MetricsAsync();
+            CheckMetrics(metrics);
+        }
+
+        [Fact]
+        public async Task MetricsEventFiredOnConsoleTimespan()
+        {
+            var metricsTaskWrapper = new TaskCompletionSource<MetricEventArgs>();
+            Page.MetricsReceived += (sender, e) => metricsTaskWrapper.SetResult(e);
+
+            await Page.EvaluateExpressionAsync("console.timeStamp('test42')");
+            var result = await metricsTaskWrapper.Task;
+
+            Assert.Equal("test42", result.Title);
+            CheckMetrics(result.Metrics);
+        }
+
+        private void CheckMetrics(Dictionary<string, decimal> metrics)
+        {
+            var metricsToCheck = new List<string>{
+                "Timestamp",
+                "Documents",
+                "Frames",
+                "JSEventListeners",
+                "Nodes",
+                "LayoutCount",
+                "RecalcStyleCount",
+                "LayoutDuration",
+                "RecalcStyleDuration",
+                "ScriptDuration",
+                "TaskDuration",
+                "JSHeapUsedSize",
+                "JSHeapTotalSize",
+            };
+
+            foreach (var name in metrics.Keys)
+            {
+                Assert.Contains(name, metricsToCheck);
+                Assert.True(metrics[name] >= 0);
```

Comment:
@Meir017 this looks like a dumb `Assert` to me. Puppeteer might be doing that to check if it has a number. We enforce that because we are using a real language. 
Would you leave it or remove it?

Reply:
I guess I would leave it until we refactor the metrics to a strongly typed class